shell/platform/linux/fl_mouse_cursor_plugin.cc

```diff
@@ -14,6 +14,12 @@ static constexpr char kBadArgumentsError[] = "Bad Arguments";
 static constexpr char kActivateSystemCursorMethod[] = "activateSystemCursor";
 static constexpr char kKindKey[] = "kind";
 
+static GHashTable* systemCursorTable = nullptr;
```

Comment:
It would be better to put this inside the `FlMouseCursorPlugin` struct, so then it can be removed in `fl_mouse_cursor_plugin_dispose`. The table would then be created and populated in `fl_mouse_cursor_plugin_init`. Also the naming convention here would be `system_cursor_table`.